crest/Assets/Crest/Crest/Scripts/LodData/RegisterClipSurfaceInput.cs

```diff
@@ -78,6 +82,41 @@ public enum Primitive
         // The clip surface samples at the displaced position in the ocean shader, so the displacement correction is not needed.
         protected override bool FollowHorizontalMotion => true;
 
+        #region Painting
+        [Header("Paint Settings")]
+        public CPUTexture2DPaintable_R16_AddBlend _paintData;
```

Comment:
the data used by the input is defined by this concrete derived type from CPUTexture